src/client/public-openfin/rt-fx.json

```diff
@@ -22,12 +23,7 @@
     "version": "<OPENFIN_RUNTIME>",
     "arguments": "--remote-debugging-port=9091"
   },
-  "services": [
-    {
-      "name": "notifications",
-      "manifestUrl": "https://cdn.openfin.co/services/openfin/notifications/1.20.0/app.json"
-    }
-  ],
+  "services": [{ "name": "notifications" }],
```

Comment:
incidental - just keeps us up to date with Notifications, I don't think we need to peg it to a particular version any more (per OpenFin advice) .. but open to discussion on this